Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -683,27 +715,30 @@ window.__firefox__.execute(function($) {
 
   /**
    * Extract any selectors from the document
-   * @param {*} switchToMutationObserverAtTime A timestamp that identifies when we should switch to the mutation observer
+   * @param {object} element The element to extract the selectors from
    */
-  const querySelectorsFromElement = async (element, switchToMutationObserverAtTime) => {
+  const querySelectorsFromElement = (element) => {
     extractNewSelectors(element)
-    const elmWithClassOrId = element.querySelectorAll('[class],[id]')
+    const elmWithClassOrId = element.querySelectorAll(classIdWithoutHtmlOrBody)
 
     elmWithClassOrId.forEach((node) => {
       extractNewSelectors(node)
     })
+  }
 
-    if (switchToMutationObserverAtTime !== undefined &&
-      window.Date.now() >= switchToMutationObserverAtTime) {
-      useMutationObserver()
-    }
+  /**
+   * Extract any selectors from the document
+   * @param {number} switchToMutationObserverAtTime A timestamp indicating when we shoudl switch back to mutation observer
+   */
+  const querySelectorsFromDocument = (switchToMutationObserverAtTime) => {
+    querySelectorsFromElement(document)
 
-    const changes = await sendPendingSelectorsIfNeeded()
-    if (!changes) { return }
-    setRulesOnStylesheet()
+    // Send any found selectors to the browser
+    sendPendingSelectorsThrottled()
 
-    if (!args.hideFirstPartyContent) {
-      pumpCosmeticFilterQueuesOnIdle()
+    if (switchToMutationObserverAtTime !== undefined &&
+      window.Date.now() >= switchToMutationObserverAtTime) {
+      useMutationObserver('CALLBACK')
```

Comment:
This function doesn't accept any parameters. Why do we pass this string as a parameter?

Reply:
Sorry I was adding that for testing and accidentally merged it. Ill fix it

Reply:
Done